tests/unit/records/marshmallow/literature/common/test_accelerator_experiment.py

```diff
@@ -124,3 +125,25 @@ def test_returns_dashed_institution_accelerator_experiment_as_name_with_unicode(
     result = schema.dumps(record).data
 
     assert expected == json.loads(result)
+
+
+@mock.patch("inspirehep.records.api.base.InspireRecord.get_records_by_pids")
+def test_accelerator_experiment_many(get_records_mock):
```

Comment:
I'd move this to an integration test :)

Reply:
But why? Only thing which is tested here is one serializer, no real record is required for this, no integration with anything.

Reply:
this serializer also use db to retrieve experiments (if present), we have to actual test this case as well

Reply:
done